ui_tests/src/test/resources/com/saucedemo/account/SignInNegative.feature

```diff
@@ -0,0 +1,49 @@
+Feature: Sign In
+
+  @signIn
+  Scenario: Sign in with invalid password
+    Given I open start page
+    When I set 'problem_user' as username on Sign in page
+    And I set 'secret_juice' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
+  @signIn
+  Scenario: Sign in with invalid username
+    Given I open start page
+    When I set 'unusual_user' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
+  @signIn
+  Scenario: Sign in with invalid username and password
+    Given I open start page
+    When I set 'Iamnot_user' as username on Sign in page
+    And I set 'Idonot_care' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
+  @signIn
+  Scenario: Sign in with invalid username
+    Given I open start page
+    When I set '111_222' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
+  @signIn
+  Scenario: Sign in with invalid password
+    Given I open start page
+    When I set 'performance_glitch_user' as username on Sign in page
+    And I set '!!@@##$$' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
+  @signIn
+  Scenario: Sign In with locked out username and password
+    Given I open start page
+    When I set 'locked_out_user' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page
```

Comment:
@Firdavs11  This scenario is failing on this step. We have 2 step definitions for click on the Sign In button:
1. `I click 'Sign In' button on Sign in page`
2. `I click 'Sign In' button on Sign in page expecting failure`
Please make sure you are using the correct one

Reply:
Everything is fixed now. Please check and let me know if anything.